themes/wporg-translate-events-2024/style.css

```diff
@@ -9,3 +9,137 @@ License: GNU General Public License v2 or later
 License URI: http://www.gnu.org/licenses/gpl-2.0.html
 Text Domain: wporg-translate-events-2024
 Template: wporg-parent-2021
+*/
+
+.wporg-marker-list__container {
```

Comment:
I'm not sure why other themes named these classes like this, do you know? I mean, why "marker"? I think we could use clearer/simpler names, unless there's some convention that they are following.

Reply:
I have no idea why